core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -8723,6 +8723,61 @@ public void subTestIntervalSecondFailsValidation() {
     sql(sql3).ok(expected3);
   }
 
+  @Test public void testGroupByAlias1() {
+    conformance = SqlConformanceEnum.LENIENT;
+    final String sql = "select price,price\n"
```

Comment:
You'd better describe the difference in these tests.

Reply:
done